### PR TITLE
Ping dev

### PR DESCRIPTION
**Pull request by rvstwi**
## Summary

Adds basic rough ping functionality to the menus.
Changes variable name in OpenTimer.

## Testing

I have tested my compiled build in the following scenarios:

- [x] A client running **my build** against a server running **latest stable version**
- [ ] A client running **latest stable version** against a server running **my build**
- [ ] A client running **my build** against a server running **my build**

### CODE REVIEW

File: OpenRVS/classes/OpenMultiPlayerWidget.uc [hunk 1]
```diff
@@ -314,7 +317,7 @@ function ShowWindow()
 function Refresh(bool bActivatedByUser)
 {
 	local R6WindowListServerItem CurServer;
-
+
```
Comment (ijemafe) [inline OpenRVS/classes/OpenMultiPlayerWidget.uc:320]: extra whitespace

File: OpenRVS/classes/OpenMultiPlayerWidget.uc [hunk 2]
```diff
@@ -423,6 +433,10 @@ function ReceiveServerInfo(string sIP,coerce int iNumP,coerce int iMaxP,string s
 			else
 				CurServer.bSameVersion = true;
 			CurServer.bLocked = bSvrLocked;//1.5 added locked here
+			//1.5 add ping
+			i = Timer.EndTimer(CurServer.szIPAddr);
```
Comment (ijemafe) [inline OpenRVS/classes/OpenMultiPlayerWidget.uc:437]: For performance purposes, maybe move this above the iMaxPlayers setting? If this entire loop iteration happens in one tick then ignore me.

File: OpenRVS/classes/OpenMultiPlayerWidget.uc [hunk 3]
```diff
@@ -330,6 +333,13 @@ function Refresh(bool bActivatedByUser)
 	CurServer = R6WindowListServerItem(m_ServerListBox.GetItemAtIndex(0));
 	while ( CurServer != none )
 	{
+		//1.5 - get rough ping
+		if ( Timer == none )
+		{
+			Timer = new class'OpenTimer';//.static.New(GetEntryLevel());
```
Comment (ijemafe) [inline OpenRVS/classes/OpenMultiPlayerWidget.uc:339]: Delete comment

File: OpenRVS/classes/OpenTimer.uc [hunk 4]
```diff
@@ -22,14 +23,16 @@ const ERR_VALUE = -1;
 // Timings stores all timers in the class object
 var private array<Timing> Timings;
 //var OpenMultiPlayerWidget LevelSource;//provides clock
-var LevelInfo LevelSource;//provides clock
+var LevelInfo ClockSource;//provides clock
 
 // Custom constructor.
+//TODO - delete me
```
Comment (ijemafe) [inline OpenRVS/classes/OpenTimer.uc:29]: Can go ahead and delete this too!

File: OpenRVS/classes/OpenMultiPlayerWidget.uc (hunk 2 above)
Comment (rvstwi) [reply OpenRVS/classes/OpenMultiPlayerWidget.uc:437]: Yep all in one tick.

File: OpenRVS/classes/OpenTimer.uc (hunk 4 above)
Comment (rvstwi) [reply OpenRVS/classes/OpenTimer.uc:29]: Roger.

File: OpenRVS/classes/OpenMultiPlayerWidget.uc (hunk 1 above)
Comment (rvstwi) [reply OpenRVS/classes/OpenMultiPlayerWidget.uc:320]: My editor auto-adds tabs when you press enter so you're at the same indent level on the next line, that's all. :)

**Pull request merged**